ocs_ci/utility/utils.py

```diff
@@ -352,21 +352,44 @@ def custom_ceph_config(suite_config, custom_config, custom_config_file):
     return full_custom_config
 
 
+def mask_secrets(plaintext, secrets):
+    """
+    Replace secrets in plaintext with asterisks
+
+    Args:
+        plaintext (str): The plaintext to remove the secrets from
+        secrets (list): List of secret strings to replace in the plaintext
+
+    Returns:
+        plaintext (str): The censored version of plaintext
+
+    """
+    for secret in secrets:
+        plaintext = plaintext.replace(secret, '*' * 5)
+    return plaintext
+
+
 def run_cmd(cmd, **kwargs):
     """
     Run an arbitrary command locally
 
     Args:
         cmd (str): command to run
 
+    Keyword Args:
+        secrets (list): Whether to log the current cmd execution.
```

Comment:
Whether to log the current cmd execution. Doesn't sound like proper doc for this